.travis.yml

```diff
@@ -35,11 +35,9 @@ install:
  - true
 
 script:
-  - ./gradlew build idea -s -PcheckJava6Compatibility
+  - ./gradlew build idea -s -PcheckJava6Compatibility && ./gradlew ciPerformRelease
```

Comment:
All right, makes sense. Maybe add a comment such that future developers don't "optimize" this